10palindrome.js

```diff
@@ -0,0 +1,31 @@
+// 10. Write a function that determines if a string is a palindrome.
+var reversedString = '';
+var palindrome = function(string) {
```

Comment:
try strip out the most left and right char then pass in the middle chunk. racecar -> aceca -> cec -> e